app/cdap/components/ThemeWrapper/index.tsx

```diff
@@ -28,12 +28,14 @@ import {
   white,
   primary,
 } from 'components/ThemeWrapper/colors';
-import {
-  PaletteColor,
-  PaletteColorOptions,
-  Palette,
-  PaletteOptions,
-} from '@material-ui/core/styles/createPalette';
+import { ThemeProvider as StyledThemeProvider } from 'styled-components';
+// this isn't used?
+// import {
+//   PaletteColor,
+//   PaletteColorOptions,
+//   Palette,
+//   PaletteOptions,
+// } from '@material-ui/core/styles/createPalette';
```

Comment:
It's probably safe to delete instead of commenting - it's always in git if we need it